packages/extension-api/src/extension-api.d.ts

```diff
@@ -1320,6 +1320,149 @@ declare module '@podman-desktop/api' {
     export function createFileSystemWatcher(path: string): FileSystemWatcher;
   }
 
+  /**
+   * Content settings for a webview.
+   */
+  export interface WebviewOptions {
+    /**
+     * Root paths from which the webview can load local (filesystem) resources using uris from `asWebviewUri`
+     *
+     * Default to the root folders of the current workspace plus the extension's install directory.
+     *
+     * Pass in an empty array to disallow access to any local resources.
+     */
+    readonly localResourceRoots?: readonly Uri[];
+  }
+
+  /**
+   * Displays html content, similarly to an iframe.
+   */
+  export interface Webview {
+    /**
+     * Content settings for the webview.
+     */
+    options: WebviewOptions;
+
+    /**
+     * HTML contents of the webview.
+     *
+     * This should be a complete, valid html document. Changing this property causes the webview to be reloaded.
+     *
+     */
+    html: string;
+
+    /**
+     * Fired when the webview content posts a message.
+     *
+     * Webview content can post strings or json serializable objects back to an extension.
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    readonly onDidReceiveMessage: Event<any>;
```

Comment:
Could we use unknown instead of any ?

Reply:
yes, implementation has changed from any to unknwown but it didn't reach the API exposure
will update

Reply:
fixed